docs/en/setup/backend/configuration-vocabulary.md

```diff
@@ -202,10 +202,16 @@ core|default|role|Option values: `Mixed/Receiver/Aggregator`. **Receiver** mode
 | receiver-otel | default | A receiver for analyzing metrics data from OpenTelemetry. | - | - |
 | - | - | enabledHandlers| Enabled handlers for otel. | SW_OTEL_RECEIVER_ENABLED_HANDLERS | -                                                                       |
 | - | - | enabledOcRules| Enabled metric rules for OC handler. | SW_OTEL_RECEIVER_ENABLED_OC_RULES | -                                                                       |
-| receiver-zipkin |default|  A receiver for Zipkin traces. | - | - |
-| - | - | restHost| Binding IP of RESTful services. |SW_RECEIVER_ZIPKIN_HOST| 0.0.0.0                                                                 |
-| - | - | restPort | Binding port of RESTful services. | SW_RECEIVER_ZIPKIN_PORT| 9411                                                                    |
-| - | - | restContextPath| Web context path of RESTful services. | SW_RECEIVER_ZIPKIN_CONTEXT_PATH| /                                                                       |
+ receiver-zipkin |default|  A receiver for Zipkin traces. | - | - |
+| - | - | host| Binding IP of RESTful services. |SW_RECEIVER_ZIPKIN_HOST| 0.0.0.0                                                               |
+| - | - | port | Binding port of RESTful services. | SW_RECEIVER_ZIPKIN_PORT| 9411                                                                  |
+| - | - | contextPath| Web context path of RESTful services. | SW_RECEIVER_ZIPKIN_CONTEXT_PATH| /                                                                     |
+| - | - | maxThreads| Maximum thread number of RESTful services. | SW_RECEIVER_ZIPKIN_JETTY_MAX_THREADS| 200                                                                   |
+| - | - | idleTimeOut| Connector idle timeout of RESTful services (in milliseconds). | SW_RECEIVER_ZIPKIN_JETTY_IDLE_TIMEOUT| 30000                                                                 |
+| - | - | acceptorPriorityDelta| ServerSocketChannel backlog of RESTful services. | SW_RECEIVER_ZIPKIN_JETTY_DELTA| 0                                                                     |
+| - | - | acceptQueueSize| Maximum request header size accepted. | SW_RECEIVER_ZIPKIN_QUEUE_SIZE| 0                                                                     |
```

Comment:
Let's remove Jetty from all places. It is in v9 iteration now, we don't need this debt

Reply:
`All` includes `jetty*` in core module and all other places. These should be changed in 9.0 release, let's do this sooner than later.

Reply:
Removed all, @kezhenxu94 pls check.